app/uk/gov/hmrc/exports/services/reversemapping/declaration/DeclarationXmlParser.scala

```diff
@@ -47,7 +48,40 @@ object DeclarationXmlParser {
     }
   }
 
+  @tailrec
```

Comment:
Is this annotation for the implicit class? :thinking:

Reply:
:man_facepalming: Good spot. Forgot to remove it.